src/components/FeatureExportButton/FeatureExportButton.test.js

```diff
@@ -144,6 +144,9 @@ describe('FeatureExportButton', () => {
 
       const textStyle = new Style({
         text: new Text({
+          backgroundFill: new Fill({
+            color: 'rgba(255,255,255,0.01)',
+          }),
```

Comment:
backgroundFill is added to `<ExtendedData>` tag in the kml file.  This test only tests `<Style>`  tag . Can you add a test for ExtendedData please ?

Reply:
see https://github.com/geops/react-spatial/pull/108